<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterDialog</class>
 <widget class="QDialog" name="PrinterDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printer Properties</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Decolation Rule</string>
   </property>
  </widget>
  <widget class="QComboBox" name="decolationComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DirButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spoolDirectory">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>285</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Spool Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>33</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Port Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>153</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Stataionery</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>201</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines Per Inch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>009.009.009.009; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="stationeryComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>153</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="lpiSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>195</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>235</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>97</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectedLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnected</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>391</y>
     <width>94</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../HercStudio/HercStudio.qrc">:/icons/printer.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>380</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../HercStudio/HercStudio.qrc">:/icons/tray.xpm</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="animatedGif">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>461</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../yakov-fedora20/Desktop/connecting.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="connectedImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>461</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../HercStudio/HercStudio.qrc">:/icons/connected.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="printerNameComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>IP</tabstop>
  <tabstop>PortNumber</tabstop>
  <tabstop>stationeryComboBox</tabstop>
  <tabstop>lpiSpinBox</tabstop>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>spoolDirectory</tabstop>
  <tabstop>DirButton</tabstop>
  <tabstop>decolationComboBox</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../HercStudio/HercStudio.qrc"/>
 </resources>
 <connections/>
</ui>
